### PR TITLE
complete

### PR DESCRIPTION
**Pull request by wing2005**

### CODE REVIEW

File: CalcMyTip/app/src/main/java/com/example/calcmytip/Bill.java [hunk 1]
```diff
@@ -3,23 +3,39 @@
...
     }
 
-    public void ToTipButton(View view) {
+    public void onCalculateBill(View view) {
+
+        String billStr = editTextBill.getText().toString();
+        int bill = Integer.parseInt(billStr);
+        int tip = Integer.parseInt(tipStr);
+
+        int total = (bill*(100+tip))/100;
```
Comment (liorblob) [inline CalcMyTip/app/src/main/java/com/example/calcmytip/Bill.java:34]: can you explain the formula in a comment?

File: CalcMyTip/app/src/main/java/com/example/calcmytip/RatingBase.java [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+    public static final String KEY_TIP = "rateKey";
+
+    protected String baseTip;
+    protected String restName;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+       // setContentView(R.layout.activity_rating1);
```
Comment (liorblob) [inline CalcMyTip/app/src/main/java/com/example/calcmytip/RatingBase.java:19]: Do we need this?

**Review by liorblob — APPROVED** [review]
nice, 2 small comments.

File: CalcMyTip/app/src/main/java/com/example/calcmytip/Bill.java (hunk 1 above)
Comment (wing2005) [reply CalcMyTip/app/src/main/java/com/example/calcmytip/Bill.java:34]: bill - is what the user entered.
tip is the calculated tip (from 0-18)
basic percentage formula....

File: CalcMyTip/app/src/main/java/com/example/calcmytip/RatingBase.java (hunk 2 above)
Comment (wing2005) [reply CalcMyTip/app/src/main/java/com/example/calcmytip/RatingBase.java:19]: we can delete the comment

**Pull request merged**